packages/tracing/src/browser/request.ts

```diff
@@ -99,12 +106,12 @@ export const defaultRequestInstrumentationOptions: RequestInstrumentationOptions
   traceFetch: true,
   traceXHR: true,
   tracingOrigins: DEFAULT_TRACING_ORIGINS,
+  tracePropagationTargets: DEFAULT_TRACING_ORIGINS,
```

Comment:
For simplicity when it comes time to delete `tracingOrigins`, can we give this its own `DEFAULT_TRACE_PROPAGATION_TARGETS` variable? That way, we won't have to change this line when we do that.